src/reducers/counter.js

```diff
@@ -0,0 +1,20 @@
+// @ts-check
+
+import { Actions } from '../actions/counter';
+
+const initialCount = 0; 
+
+export const CounterReducer = (state = { count: initialCount }, action) => { 
```

Comment:
Closer. But now it's just pulling out count. Pull out the whole initialState, because what if you wanted to add another property besides count?